corehq/apps/cloudcare/static/cloudcare/js/utils.js

```diff
@@ -233,8 +233,8 @@ hqDefine('cloudcare/js/utils', [
     };
 
     function renderMarkdown(text) {
-        const md = window.markdownit();
-        return md.render(DOMPurify.sanitize(text || "")).replaceAll("\n", "<br>");
+        const md = window.markdownit().set({ breaks: true });
```

Comment:
nit (even though this is the syntax I originally suggested):
```suggestion
        const md = window.markdownit({ breaks: true });
```